standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestFunctions.java

```diff
@@ -180,7 +180,7 @@ public void testCreateFunctionDefaultValues() throws Exception {
 
     Function createdFunction = client.getFunction(function.getDbName(),
         function.getFunctionName());
-    Assert.assertNull("Comparing OwnerName", createdFunction.getOwnerName());
+    Assert.assertEquals("Comparing OwnerName", createdFunction.getOwnerName(), createdFunction.getOwnerName());
```

Comment:
Comparing the same values (createdFunction.getOwnerName())  will obviously be true.

Reply:
ACK